tests/torch_datasets/test_process_and_combine.py

```diff
@@ -0,0 +1,219 @@
+from ocf_data_sampler.utils import minutes
+
+from ocf_data_sampler.torch_datasets.process_and_combine import (
+    process_and_combine_datasets,
+    merge_dicts,
+    fill_nans_in_arrays,
+    compute,
+)
+
+
+NWP_FREQ = pd.Timedelta("3h")
+
+
+@pytest.fixture(scope="module")
+def da_sat_like():
+    """Create dummy satellite data"""
+    x = np.arange(-100, 100)
+    y = np.arange(-100, 100)
+    datetimes = pd.date_range("2024-01-02 00:00", "2024-01-03 00:00", freq="5min")
+
+    da_sat = xr.DataArray(
+        np.random.normal(size=(len(datetimes), len(x), len(y))),
+        coords=dict(
+            time_utc=(["time_utc"], datetimes),
+            x_geostationary=(["x_geostationary"], x),
+            y_geostationary=(["y_geostationary"], y),
+        )
+    )
+    return da_sat
+
+
+@pytest.fixture(scope="module")
+def da_nwp_like():
+    """Create dummy  NWP data"""
+    x = np.arange(-100, 100)
+    y = np.arange(-100, 100)
+    datetimes = pd.date_range("2024-01-02 00:00", "2024-01-03 00:00", freq=NWP_FREQ)
+    steps = pd.timedelta_range("0h", "16h", freq="1h")
+    channels = ["t", "dswrf"]
+
+    da_nwp = xr.DataArray(
+        np.random.normal(size=(len(datetimes), len(steps), len(channels), len(x), len(y))),
+        coords=dict(
+            init_time_utc=(["init_time_utc"], datetimes),
+            step=(["step"], steps),
+            channel=(["channel"], channels),
+            x_osgb=(["x_osgb"], x),
+            y_osgb=(["y_osgb"], y),
+        )
+    )
+    return da_nwp
+
+
+@pytest.fixture
+def mock_constants(monkeypatch):
+    """Creation of dummy constants used in normalisation process"""
+    mock_nwp_means = {"ukv": {
+        "t": 10.0,
+        "dswrf": 50.0
+    }}
+    mock_nwp_stds = {"ukv": {
+        "t": 2.0,
+        "dswrf": 10.0
+    }}
+    mock_sat_means = 100.0
+    mock_sat_stds = 20.0
+    
+    monkeypatch.setattr("ocf_data_sampler.constants.NWP_MEANS", mock_nwp_means)
+    monkeypatch.setattr("ocf_data_sampler.constants.NWP_STDS", mock_nwp_stds)
+    monkeypatch.setattr("ocf_data_sampler.constants.SAT_MEANS", mock_sat_means)
+    monkeypatch.setattr("ocf_data_sampler.constants.SAT_STDS", mock_sat_stds)
+
+
+@pytest.fixture
+def mock_config():
+    """Specify dummy configuration"""
+    class MockConfig:
+        class InputData:
+            class NWP:
+                provider = "ukv"
+                interval_start_minutes = -360
+                interval_end_minutes = 180
+                time_resolution_minutes = 60
+
+            class GSP:
+                interval_start_minutes = -120
+                interval_end_minutes = 120
+                time_resolution_minutes = 30
+
+            def __init__(self):
+                self.nwp = {"ukv": self.NWP()}
+                self.gsp = self.GSP()
+
+        def __init__(self):
+            self.input_data = self.InputData()
+    
+    return MockConfig()
+
+
+@pytest.fixture
+def mock_location():
```

Comment:
Since this is only used in a single test and its just a single line of code, I think it would be cleaner just to go i that test

Reply:
Sorted and  updated